src/main/kotlin/brs/db/sql/SqlPeerDb.kt

```diff
@@ -14,9 +14,9 @@ internal class SqlPeerDb(private val dp: DependencyProvider) : PeerDb {
 
     override fun updatePeers(peers: List<String>) {
         dp.db.useDslContext { ctx ->
-            val peers = ctx.selectFrom(PEER).fetch(PEER.ADDRESS)
-            ctx.deleteFrom(PEER).where(PEER.ADDRESS.notIn(peers))
-            ctx.batch(peers.mapNotNull { if (peers.contains(it)) null else ctx.insertInto(PEER).set(PEER.ADDRESS, it) })
+            val peerEntities = ctx.selectFrom(PEER).fetch(PEER.ADDRESS)
```

Comment:
Could you perhaps use `dbPeers` as a name instead? It makes it clearer what the value actually is